notebooks/multimodal_intro.ipynb

```diff
@@ -8,7 +8,7 @@
       "source": [
```

Comment:
<p>We don't need this info note here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/248/discussion/#comment-2235967969'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
removed